test/format/ops.c

```diff
@@ -226,14 +246,22 @@ rollback_to_stable(WT_SESSION *session)
 
     /* Check the saved snap operations for consistency. */
     snap_repeat_rollback(session, tinfo_list, GV(RUNS_THREADS));
+
+    /*
+     * For a predictable run, the final stable timestamp is known and fixed, but individual threads
+     * may have gone beyond that. Now that we've rolled back, set the current timestamp to the
+     * stable so that next run starts from a known value.
+     */
+    if (GV(RUNS_PREDICTABLE_REPLAY))
+        g.timestamp = g.stable_timestamp;
 }
 
 /*
  * operations --
  *     Perform a number of operations in a set of threads.
  */
 void
-operations(u_int ops_seconds, bool lastrun)
+operations(u_int ops_seconds, u_int run_number, u_int run_count)
```

Comment:
`run_number` and `run_count` are pretty similarly named to each other. Could you find one or two different names that makes them more distinctive.

Reply:
Did you think about this? To me `count` and `number` have a lot of potentially overlapping meaning.